bundle/src/lib/header-bidding/prebid/prebid.ts

```diff
@@ -410,6 +416,35 @@ const initialise = async (
 		}
 	}
 
+	if (getConsentFor('uid2', consentState)) {
```

Comment:
I need to update in the `libs` repo is require to add the `uid2` vendor name.

node_modules/.pnpm/@guardian+libs@26.0.1_@guardian+ophan-tracker-js@2.6.1_tslib@2.8.1_typescript@5.9.3/node_modules/@guardian/libs/dist/consent-management-platform/vendors.d.ts

Reply:
@dskamiotis as discussed you shouldn't need to add the vendor to the list, as:

1) It should already be present, I can see the "The UK Trade Desk Ltd" is in the current vendor list.
2) The Identity & Trust team are responsible for managing the vendor list, so if it wasn't present we would need to work with them to get it added.

Reply:
We only want to integrate `uid2` in the USA, to check US users consent state you can read `consentState.usnat.doNotSell` - if this is `true` we should **not** integrate. 

I'm not sure what `consentState.usnat.doNotSell` resolves to for users not in the US, I'd recommened testing this. 

We don't want to integrate _unless_ the user is in the US and `consentState.usnat.doNotSell` is `false`.